assets/src/edit-story/elements/text/display.js

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
This file is a bit long, but I don't see how it can be easily compressed. It's fairly well-contained IMHO, but maybe some of the keyboard click handling can be moved to a helper hook.

Reply:
Actually some of the keyboard stuff will be removed when `<Focusable />` is included here. Optimizations will wait for then.